pom.xml

```diff
@@ -534,20 +534,9 @@
             <version>2.11</version>
             <configuration>
               <header>LICENSE-short.txt</header>
-              <excludes>
-                <exclude>**/README</exclude>
-                <exclude>**/*.yml</exclude>
-                <exclude>*.xml</exclude>
-                <exclude>*.txt</exclude>
-                <exclude>*.md</exclude>
-                <exclude>doc/**</exclude>
-                <exclude>spooned/**</exclude>
-                <exclude>src/main/resources/**</exclude>
-                <exclude>src/main/assembly/**</exclude>
-                <exclude>src/test/**</exclude>
-                <exclude>src/changes/**</exclude>
-                <exclude>src/site/**</exclude>
-              </excludes>
+              <includes>
+                <include>src/main/java/**</include>
```

Comment:
`package.html` files already have license headers (e.g., [package.html](https://github.com/INRIA/spoon/blob/master/src/main/java/spoon/package.html)). So, why not?